map_server/src/image_loader.cpp

```diff
@@ -43,7 +43,7 @@
 #include <SDL/SDL_image.h>
 
 #include "map_server/image_loader.h"
-#include <tf/tf.h>
+#include <tf2/LinearMath/Quaternion.h>
```

Comment:
I think I'll continue to abuse tf2's internal implementation here because I don't want to add a dependency on bullet to ROS 2 right now.